src/main/java/ru/yandex/practicum/filmorate/storage/UserStorage.java

```diff
@@ -0,0 +1,17 @@
+package ru.yandex.practicum.filmorate.storage;
+
+import org.springframework.web.bind.annotation.RequestBody;
+import ru.yandex.practicum.filmorate.model.User;
+
+import javax.validation.Valid;
+import java.util.Collection;
+
+public interface UserStorage {
+    public Collection<User> findAll();
```

Comment:
```suggestion
    List<User> findAll();
```
Нужно убрать `public` в методах интерфейса